recipes/dftd3-python/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "dftd3-python" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/awvwgk/simple-dftd3/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 79b507a0d892f19222e1403f4c7ebb98ce9eb96b3b8d4821ba5631aa9ac756be
+
+build:
+  number: 0
+  skip: True  # [win or (osx and (python_impl == 'cpython' and py<38))]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - meson 0.55.*
+    - python
+    - cffi
+    - pkgconfig
+  host:
+    - python
+    - pip
+    - simple-dftd3
+    - libblas
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux and not aarch64]
```

Comment:
What is the cause of not needing `libgomp` on `aarch64`?

Reply:
Found it in the maintainer knowledge base originally this way, I see it was removed from there a while ago: https://github.com/conda-forge/conda-forge.github.io/pull/976/commits/aea88f56f0d05705c731407c0bc983efbc95474a. I just tried running with OpenMP dependency and this seems to work fine for the CFFI module.